cratedb_toolkit/cfr/cli.py

```diff
@@ -127,6 +128,40 @@ def job_statistics_view(ctx: click.Context):
     jd(response)
 
 
+@make_command(job_statistics, "report", "View job statistics per report.")
+@click.pass_context
+def job_statistics_report(ctx: click.Context):
+    """
+    View job statistics about collected queries per report.
+    """
+    import cratedb_toolkit.cfr.marimo
+
+    address = DatabaseAddress.from_string(ctx.meta["cratedb_http_url"] or ctx.meta["cratedb_sqlalchemy_url"])
+    os.environ["CRATEDB_SQLALCHEMY_URL"] = address.dburi
+    cratedb_toolkit.cfr.marimo.app.run()
+
+
+@make_command(job_statistics, "ui", "View job statistics per web UI.")
+@click.pass_context
+def job_statistics_ui(ctx: click.Context):
+    """
+    View job statistics about collected queries per web UI.
+    """
+    import marimo
+    import uvicorn
+    from fastapi import FastAPI
+
+    import cratedb_toolkit.cfr.marimo
+
+    address = DatabaseAddress.from_string(ctx.meta["cratedb_http_url"] or ctx.meta["cratedb_sqlalchemy_url"])
+    os.environ["CRATEDB_SQLALCHEMY_URL"] = address.dburi
+    server = marimo.create_asgi_app()
+    server = server.with_app(path="/", root=cratedb_toolkit.cfr.marimo.__file__)
+    app = FastAPI()
+    app.mount("/", server.build())
+    uvicorn.run(app, host="localhost", port=7777, log_level="info")
```

Comment:
Dear @WalBeh, this is how you can "embed" a Marimo application into a regular Python program, in simple variant. Cheers, Moll.